packages/storybook/src/nimble/tabs/tabs.stories.ts

```diff
@@ -42,18 +44,58 @@ const metadata: Meta<TabsArgs> = {
     }
 };
 
+const simpleTabs: readonly TabArgs[] = [
+    { title: 'Tab 1', id: '1', disabled: false },
+    { title: 'Tab 2', id: '2', disabled: true },
+    { title: 'Tab 3', id: '3', disabled: false }
+] as const;
+
+const wideTabs: readonly TabArgs[] = [
+    {
+        title: 'Option 1 that is too long to fit in the drop down width',
+        id: '0',
+        disabled: false
+    },
+    {
+        title: 'Option 2 that is also too long but disabled',
+        id: '1',
+        disabled: true
+    },
+    { title: 'Short', id: '2', disabled: false }
+] as const;
+
+const manyTabs: TabArgs[] = [];
+for (let i = 0; i < 100; i++) {
+    manyTabs.push({
+        title: `Option ${i}`,
+        id: `${i}`,
+        disabled: false
+    });
+}
+
+const tabSets = {
+    [ExampleTabsType.simpleTabs]: simpleTabs,
+    [ExampleTabsType.wideTabs]: wideTabs,
+    [ExampleTabsType.manyTabs]: manyTabs
+} as const;
+
 export default metadata;
 
 export const tabs: StoryObj<TabsArgs> = {
     // prettier-ignore
     render: createUserSelectedThemeStory(html`
         <${tabsTag} activeid="${x => x.activeid}">
-            <${tabTag} id="1" disabled">Tab One</${tabTag}>
-            <${tabTag} id="2">Tab Two</${tabTag}>
-            <${tabTag} id="3">Tab Three</${tabTag}>
-            <${tabPanelTag}>Content of the first tab</${tabPanelTag}>
-            <${tabPanelTag}>Content of the second tab</${tabPanelTag}>
-            <${tabPanelTag}>Content of the third tab</${tabPanelTag}>
+        ${repeat(x => (tabSets[x.tabsType] as TabArgs[]), html<TabArgs>`
```

Comment:
Seems like there is a visual difference when activeid is set to a disabled tab in anchor tabs vs tabs. Looks like anchor tabs has a disabled state visual

Reply:
We discussed this off-line, and this is an existing distinction between the two. Ultimately, for the `Tabs` component we want to display the underline on a disabled, active tab. We will capture his in a separate task, as this will require more significant changes.